pkg/deploy/elbv2/listener_manager.go

```diff
@@ -72,19 +80,22 @@ func (m *defaultListenerManager) Create(ctx context.Context, resLS *elbv2model.L
 	return buildResListenerStatus(sdkLS), nil
```

Comment:
let's change buildResListenerStatus to be consistent with others like buildResListenerRuleStatus, which accepts a ListenerWithTags instead of `*elbv2model.Listener`
and construct sdkLS as 
```
sdkLS := ListenerWithTags{
		TargetGroup: resp. Listeners[0],
		Tags:        lsTags,
	}
```

Same for other functions that accepts a sdkLS (make all sdkLS to be of type ListenerWithTags)

Reply:
ok